pkg/ddc/base/runtime.go

```diff
@@ -171,6 +173,13 @@ func BuildRuntimeInfo(name string,
 
 type RuntimeInfoOption func(info *RuntimeInfo)
 
+func GetMetadataListFromAnnotation(accessor metav1.ObjectMetaAccessor) (ret []datav1alpha1.Metadata) {
+	annotations := accessor.GetObjectMeta().GetAnnotations()
```

Comment:
I think `json.Unmarshal()` can have some CPU overheads, especially if it is called frequently or used to parse large amounts of data. I have the following suggestions:

1. Checking if the specified metadata is empty. If it is, there is no need to execute the further parsing operation.
2. Checking if  the presence of the data.fluid.io/metadataList key exist. If it isn't,  there is no need to execute the further parsing operation.
3. Checking the type of metadata. If the type is incorrect, the parsing operation cannot be completed. In this case, an error message should be logging at least. if the metadata is of the wrong type, no one will be aware of this.

Reply:
I'll do the first and second steps, but in the third step, which log should I use?